library/alloc/src/vec/mod.rs

```diff
@@ -3025,26 +3025,29 @@ impl<T: Clone, A: Allocator> Vec<T, A> {
         self.spec_extend(other.iter())
     }
 
-    /// Copies elements from `src` range to the end of the vector.
+    /// Copies elements from one part of the vector to the end.
     ///
-    /// # Panics
```

Comment:
`# Panics` is a standard heading at this point. I'd retain this. See https://rust-lang.github.io/rust-clippy/master/index.html#missing_panics_doc